### ISSUE TITLE
Results reproducibility

### ISSUE THREAD

**Issue by jacopocavazza** [kracwarlock/action-recognition-visual-attention#13]
Hi @kracwarlock! Thank you for sharing your code of your amazing paper! In order to reproduce your published results, I was wondering how to select the validation split for HMDB-51 and Hollywood2 datasets. Referring to the latter ones, can you please share your files valid_labels.txt, train_labels.txt, test_labels.txt, train_filenames.txt, test_filenames.txt and valid_filenames.txt for that two datasets? I will really appreciate it a lot :) :) :)

**Comment by kracwarlock** [comment]
[HMDB-51](http://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/) provides three train-test splits . We just used split 1. Hollywood2 also provides a train-test split. For validation we used 15% of the training data as validation set and the rest 85% for training. We noted the cost at best performance on the validation set. We then trained on the entire training data until the cost reached the same value as noted with the validation set separated.
Will share the txt files.

**Comment by WeihongM** [comment]
@kracwarlock  Hi, also hope you can share your txt files, Thanks

**Comment by mpkuse** [comment]
I am trying to figure out how was the 1st attention initialized. In the attached figure, l1? 


![image](https://user-images.githubusercontent.com/10494231/47280770-8b4e9480-d60a-11e8-846e-944878e618b3.png)
